wpeverest/everest-forms
# Fix - Form Templates Ajax Filter

### All Submissions:






### Changes proposed in this Pull Request:

This PR solves the bug of Form Template Tab filter. Previously, the plan name was not showing properly. 

### How to test the changes in this Pull Request:

1. Go to the Add New Option of Everest Forms.
2. Filter the pre build templates with the help of tabs i.e. All, Free and Premium.
3. Now see the plan name. It will show the correct plan name.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Form Template plan name while filtering by tabs
